tests/test_compression.py

```
@@ -0,0 +1,244 @@
+
+# compress: happy path
+
+
+def test_compress_happy_path_returns_compressed() -> None:
+    text = "x" * 1000
+    fake = MagicMock()
+    result = MagicMock()
+    result.compressed = "compressed"
+    fake.compress.return_value = result
+    with (
+        patch.object(compression, "_HEADROOM_AVAILABLE", True),
```

Ruff (0.16.3)

[warning] Boolean positional value in function call

(FBT003)